src/core/agent/services/ipexCommunicationService.ts

```diff
@@ -974,6 +972,117 @@ class IpexCommunicationService extends AgentService {
       status: CredentialStatus.PENDING,
     };
   }
+
+  async multisigAdmit(
```

Comment:
Actually, this can now be private and just covered by the tests for `acceptAcdc` and `acceptAcdcFromMultisigExn`. I think it should cover all the scenarios by default (`acceptAcdc` will create the admit and `acceptAcdcFromMultisigExn` with pass `adminExnToJoin`

Reply:
Some test cases of acceptAcdc() still call to multisigAdmit(). If i change multisigAdmit() to private, these test cases will not work

Reply:
The tests for `acceptAcdc` should just call `acceptAcdc`, which will under the hood call `this.multisigAdmit` - that will work even if it's private.

If it's not used outside this class, it should be private and tested via the functions that call it.

Reply:
If it doesn't work, then there's something wrong in the test setup or mocking. We should only be mocking things external to the class.

Reply:
@Sotatek-TungNguyen2a This still is not private.